src/routes/results.js

```diff
@@ -1,32 +1,34 @@
 import React from 'react';
-import { Button, StyleSheet, Text, View, Image } from 'react-native';
+import { Button, StyleSheet, View, Image } from 'react-native';
 
 import colors from '../assets/colors.js'
 import images from '../assets/images.js'
-import metrics from '../themes/metrics.js'
 
 import Navbar from '../components/Navbar.js'
 import FullWidthContainer from '../components/FullWidthContainer.js'
+import {standardStyle} from '../assets/styles.js'
 import LargeText from '../components/LargeText.js'
 
 class Results extends React.Component {
   render() {
     return ( //The info here is hardcoded, but we will enevtually want to make the info change depending on the item scanned.
-      <View
-        style={styles.results}>
-        <Text style={styles.headerUpper}>How to recycle</Text>
-        <Text style={styles.headerLower}>Beverage cans</Text>
-        <FullWidthContainer flex={containerFlex}>
-          <View style={{flexDirection: 'column'}}>
-            <Image source={images.recycleSymbol}/>
-            <Text>In recycling</Text>
-          </View>
-          <Text>Empty soda and bevrage cans should be rinsed first, then put in recycling</Text>
-        </FullWidthContainer>
-        <Button
-          title='Go To Landing'
-          onPress={() => this.props.navigation.navigate('Landing')}
-        />
+      <View style = {standardStyle.wrapper}>
```

Comment:
The extra spaces in this file: `style = {` etc make it slightly harder to read. I should look into an autoformatter so robots can help standardize.

Reply:
Interesting, I always thought that having spaces around the equals made it easier to read.

Reply:
ah, in regular code, yes. I realize now that the style for jsx must be inherited from html where attributes are written without spaces `<div class="classname">`

Reply:
So, spaces in javascript, no spaces in jsx (even though the jsx is inside of javascript) :D